### PR TITLE
Gives the mime and cent-com official stamps.

### PR DESCRIPTION
**Pull request by azzzertyy**
# General Documentation

### Intent of your Pull Request

Gives the mime his own stamp in his bag.
Gives cent-com official their own stamps in their bags.

large_stamp-mime

large_stamp-cent

### Why is this change good for the game?

Clown has too much content. Share some with mime.
Cent-com stamp was partly already there, just undefined so I may swell finish it while I'm here.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds mime and cc stamp

### What should players be aware of when it comes to the changes your PR is implementing?
The mime stamp is for all intents and purposes useless.

### What general grouping does this PR fall under? 
Mime Stuff & CC Stuff

# Changelog

:cl:  
rscadd: Added Stamps for mime & cco
imageadd: Added Stamp pngs for mime & cco
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
i love it

the changelog is bloated though

**Comment by azzzertyy** [comment]
> i love it
> 
> the changelog is bloated though

cut me some slack its my first pr

**Comment by Bibby0110** [comment]
> > i love it
> > the changelog is bloated though
> 
> cut me some slack its my first pr

No need to get defensive my guy
